packages/components/src/chart-uplot/index.tsx

```diff
@@ -86,6 +88,7 @@ export default function UplotChart( {
 						ticks: {
 							show: false,
 						},
+						filter: yAxisFilter,
```

Comment:
I tried the [axis option incrs](https://github.com/leeoniya/uPlot/issues/786#issuecomment-1381245033), and it seems we can get the same effect by setting `incrs: [1, 2, 5, 10, 20, 25, 50]`.

Reply:
ha that'll be awesome!

Reply:
Wow, nice find @dognose24 !